ftfy/badness.py

```diff
@@ -118,20 +118,56 @@ def _make_weirdness_regex():
 # These are sequences that are common mojibake, resulting from common encodings
 # that are mixed up with UTF-8 on characters from their own character map.
 #
+# Sequences that match this regex will increase the `sequence_weirdness` of a
+# string.
+#
 # This helps to strengthen evidence that text should be fixed in a way that's
 # separate from the character classes above, or to counter COMMON_SYMBOL_RE's
 # fondness for characters such as inverted exclamation marks and multiplication
 # signs in contexts where they really do look like mojibake.
 
 MOJIBAKE_SYMBOL_RE = re.compile(
-    # Mojibake of low-numbered characters from ISO-8859-1 and, in some cases,
-    # ISO-8859-2. This also covers some cases from related encodings such as
-    # Windows-1252 and Windows-1250.
-    '[ÂÃĂ][\x80-\x9f€ƒ‚„†‡ˆ‰‹Œ“•˜œŸ¡¢£¤¥¦§¨ª«¬¯°±²³µ¶·¸¹º¼½¾¿ˇ˘˝]|'
+    # If the first character is in [ÂÃĂ], this covers mojibake of low-numbered
+    # characters from ISO-8859-1 and, in some cases, ISO-8859-2. This also
+    # covers some cases from related encodings such as Windows-1252 and
+    # Windows-1250.
+    # 
+    # The codepoints of decoded characters these correspond to are as follows:
+    #
+    #   Initial char.   Codepoints  What's in this range
+    #   -------------   ----------  --------------------
+    #   Â               80..BF      Latin-1 control characters and symbols
+    #   Ã, Ă            C0..FF      Latin-1 accented letters
+    #   Î               380..3BF    Greek letters
+    #   Ï, Ď            3C0..3FF    Greek letters
+    #   Ð, Đ            400..43F    Cyrillic letters
+    #   Ñ, Ń            440..47F    Cyrillic letters
+    #   Ø, Ř            600..63F    Arabic letters
+    #   Ù, Ů            640..67F    Arabic letters
+    #
+    # Here, we leave out Hebrew letters (which have a separate heuristic), and
+    # the rarer letters from each alphabet. This regex doesn't have to contain
+    # every character we want to decode -- just the sequences that we want to
+    # specifically detect as 'this looks like mojibake'.
+    # 
+    # We avoid ranges whose mojibake starts with characters like Ó -- which
+    # maps to Armenian and some more Cyrillic letters -- because those could be
+    # the 'eyes' of kaomoji faces.
+    #
+    # The set of possible second characters covers symbols that are unlikely to
+    # have a real meaning when following one of these capital letters, and appear
+    # as the second character in Latin-1, Windows-1250, or Windows-1252 mojibake.
+    '[ÂÃÎÏÐÑØÙĂĎĐŃŘŮ][\x80-\x9f€ƒ‚„†‡ˆ‰‹Œ“•˜œŸ¡¢£¤¥¦§¨ª«¬¯°±²³µ¶·¸¹º¼½¾¿ˇ˘˝]|'
```

Comment:
I've looked over the PR and everything looks right to me.  Just as a matter of due diligence: I noticed that the second half of this line didn't change.  Having extended to Greek/Cyrillic/Arabic, are there any additional characters that might now appear as the second character?  (I'd certainly believe that the answer is "no, these are the non-alphabetic characters that you can get, regardless of what the first character is", but I felt like I should check.)

Reply:
the set of 64 characters that you can get as the second byte of UTF-8 mojibake is always the same, and the set that was reasonable to include after a capital accented letter hasn't changed, as far as I know. (please don't inform me of a word in some language that contains "ŘŸ")

(no, if that word really exists, please do inform me)

Reply:
I suspect it does not.  OK, I'm convinced--like I said, wanted to be certain.